AzureCalling/Controllers/datasource/ParticipantListDataSource.swift

```diff
@@ -12,20 +12,25 @@ struct ParticipantInfo {
 
 typealias ParticipantsFetcher = () -> [ParticipantInfo]
 
-class ParticipantListDataSource: NSObject, UITableViewDelegate, UITableViewDataSource {
+class ParticipantListDataSource: NSObject, BottomDrawerDataSource {
 
     // MARK: Properties
 
     private var participantList = [BottomDrawerItem]()
     private var participantsFetcher: ParticipantsFetcher
 
+    // MARK: Initialization
+
     init(participantsFetcher: @escaping ParticipantsFetcher) {
         self.participantsFetcher = participantsFetcher
+
+        super.init()
```

Comment:
Why don't we put `super.init()` before assigning participantsFetcher? Is it because alert here?

Reply:
It could be that `participantsFetcher` is non-optional, hence it needs to be initialized prior to calling `super.init()`